examples/riot/blinky/main.c

```diff
@@ -0,0 +1,51 @@
+#include "board.h"
+#include "reactor-uc/reactor-uc.h"
+
+typedef struct {
```

Comment:
if you rebase master you can use the new macro shorthands.

Reply:
Done